source/modules/cms_vehicle_simulator/source/tests/infrastructure/__snapshots__/test_snapshot/test_cms_vehicle_simulator_snapshot.json

```diff
@@ -5360,7 +5360,7 @@
         "ssmappuniqueidregistermodule9C5C2C5D"
       ],
       "Properties": {
-        "Create": "{\"action\":\"describeEndpoint\",\"service\":\"Iot\",\"parameters\":{\"endpointType\":\"iot:Data-ATS\"},\"physicalResourceId\":{\"responsePath\":\"endpointAddress\"}}",
+        "Create": "{\"action\":\"describeEndpoint\",\"service\":\"Iot\",\"parameters\":{\"endpointType\":\"iot:Data-ATS\"},\"physicalResourceId\":{\"responsePath\":\"endpointAddress\"},\"logApiResponseData\":true}",
```

Comment:
is this correct?

Reply:
cdk version upgrade added a field to a construct used in a few places.